felles/src/main/kotlin/no/nav/familie/kontrakter/felles/klage/KlagebehandlingDto.kt

```diff
@@ -13,4 +13,12 @@ data class KlagebehandlingDto(
     val resultat: BehandlingResultat?,
     val årsak: Årsak?,
     val vedtaksdato: LocalDateTime?,
+    val eksternKlageresultat: List<EksternKlageresultatDto> = emptyList()
```

Comment:
Burde vi kalle den for noe annet enn "ekstern" her, føler ikke helt den gir noen informasjon om hva som menes.
Kan vi kallle den for `klageresultat` eller noe annet?

Reply:
Blir så fort vanskelig å skille mellom resultat og klageresultat. Følte veldig på det når vi skulle jobbe med disse to i UI, altså vårt (behandlings-resultat)resultat og kabalresultat som vises litt om hverandre. Men er ikke fornøyd med ekstern, så tar gjerne imot flere forslag :) 

Hvis jeg ikke husker feil så skal vi gå over til å bruke "klageinstans" i ui, så kanskje klageinstansResultat? 

Har også hatt litt lyst til å endre resultat til behandlingResultat eller liknende, men ikke i denne PR'en 😛

Reply:
Endrer til "klageinstans". Dropper ekstern!